xml/System.Net.Http/HttpClientHandler.xml

```diff
@@ -656,8 +656,8 @@ After NuGet package v4.3.2, the default value of <xref:System.Net.DecompressionM
         <ReturnType>System.Security.Authentication.SslProtocols</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets the SSL protocol used by the HttpClient objects managed by the HttpClientHandler object.</summary>
+        <value>One of the values defined in the SecurityProtocolType enumeration.</value>
```

Comment:
"SecurityProtocolType" is not correct.  Actually, this property is of type "SslProtocols".  So, it takes values from that enumeration.

There is a subtle relationship between the types: `SecurityProtocolType` and `SslProtocols`.  For example, the ServicePointManager property that is similar to this is called SecurityProtocols.